src/sx/README.md

```diff
@@ -61,17 +61,17 @@ export default class MyDocument extends Document {
     return { ...html, css };
   }
 
-  render(): React.Node {
+  render() {
     return (
-      <Html>
+      <html>
```

Comment:
I think this should stay as `Html` [next-custom-document](https://nextjs.org/docs/advanced-features/custom-document)

Reply:
I see, so add import from `next/document` maybe?

Reply:
Could be, yes - it's a Next.js example so it makes sense.